app/controllers/organizations/invitations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Organizations::InvitationsController < Devise::InvitationsController
+  before_action :require_organization_admin, only: [:new, :create]
+  layout "dashboard", only: [:new, :create]
+
+  def new
+    @user = User.new
+    @staff = StaffAccount.new(user: @user)
+  end
+
+  def create
+    @user = User.new(user_params.merge(password: SecureRandom.hex(8)).except(:staff_account_attributes))
+    @user.staff_account = StaffAccount.new(verified: true)
+
+    if @user.save
+      @user.staff_account.add_role(user_params[:staff_account_attributes][:roles])
+      @user.invite!(current_user)
+      redirect_to staff_index_path, notice: "Staff saved successfully."
```

Comment:
Can we chance to something like "Invite sent!"?

Reply:
I think both make sense here. The staff account does get saved and the invitation is sent

Reply:
I think "Invite sent!" makes more sense from the user perspective. Fixed!

I also changed texts in a couple of other places from "Add staff" to "Invite staff".